names.txt

```diff
@@ -22,3 +22,4 @@ List of names:
 20. Harindra
 21. Burhanuddin Rangwala
 22. Sitaram Koundinya
+23. Amit.Karnam
```

Comment:
Looks good @AmitKarnam 

Can you please remove the period symbol between your name?